.rubocop.yml

```diff
@@ -39,6 +39,9 @@ Lint/StructNewOverride:
 
 Rails:
   Enabled: true
+  Exclude:
+    - app/services/onboarding_csv_reader.rb
```

Comment:
Rude 😂 all kidding aside, I don't want to exclude these because I expect them to conform to the same behavior as the rest of the app, what issues were you getting?